kura/org.eclipse.kura.linux.net/META-INF/MANIFEST.MF

```diff
@@ -9,10 +9,12 @@ Service-Component: OSGI-INF/*.xml
 Bundle-ClassPath: .
 Bundle-ActivationPolicy: lazy
 Import-Package: javax.microedition.io,
- org.apache.commons.io;version="2.4.0",
+ org.apache.commons.exec;version="[1.3.0,2.0.0)",
+ org.apache.commons.io;version="[2.4.0,3.0.0)",
+ org.apache.commons.io.output;version="[2.4.0,3.0.0)",
  org.eclipse.kura;version="[1.1,2.0)",
  org.eclipse.kura.comm;version="[1.0,2.0)",
- org.eclipse.kura.core.linux.util;version="[1.0,2.0)",
+ org.eclipse.kura.core.linux.util;version="1.0.0",
```

Comment:
The changes seem to be ok, but I had no time to test that effectively those work. I'll do that on Monday when our offices will be re-opened.